src/test/librbd/managed_lock/test_mock_ReleaseRequest.cc

```diff
@@ -17,7 +19,36 @@ struct Traits<MockImageCtx> {
   typedef librbd::MockImageWatcher Watcher;
 };
 }
-}
+
+namespace managed_lock {
+
+template <>
+struct BreakLockRequest<librbd::MockImageCtx> {
```

Comment:
This is because this file includes ManagedLock.cc and these methods are used there (in my implementation).